src/front/app/features/challenge/challenge.reducer.js

```diff
@@ -12,8 +14,12 @@ const reducer = (state = initialState, action = {}) => {
         return upsertChallenge(action.challenge, state);
     case RESET_STATE:
         return initialState;
-    case GET_USER:
+    case USER_RECEIVED:
         return upsertCurrentUser(action.user, state);
+    case ACCEPTED_RECEIVED:
```

Comment:
@borys-ihnatyev, we can but I don't see any reason why we should. Firstly, `isWatched` & `isAccepted` are attributes of the challenge - the one thing this state is responsible for. Secondly, is it wise to spam global state with a sub-state containing one boolean value?

Could you please provide more details on this if I'm missing something?